docs/api/1.x/resource.rst

```diff
@@ -427,18 +200,6 @@ A ``details`` attribute in the response provides the offending record and
 field name. See :ref:`dedicated section about errors <error-responses>`.
 
 
-Timestamp
----------
-
-When a record is created, the timestamp of the collection is incremented.
-
-It is possible to force the timestamp if the specified record has a
-``last_modified`` field.
-
-If the specified timestamp is in the past, the collection timestamp does not
-take the value of the created record but is bumped into the future as usual.
-
-
```

Comment:
I am truely sorry if you had explained it already :( I usually review from the github UI and don't read the commit messages... my fault.

Regarding the change itself, shouldn't we add link like `For more info about timestamps behaviour, check out...` ? (genuine question, too much links can sometimes be confusing)